oauthenticator/bitbucket.py

```diff
@@ -40,11 +48,13 @@ def _authorize_url_default(self):
     def _token_url_default(self):
         return "https://bitbucket.org/site/oauth2/access_token"
 
-    team_whitelist = Set(
-        config=True, help="Automatically whitelist members of selected teams"
+    team_whitelist = Set(help="Deprecated, use `BitbucketOAuthenticator.allowed_teams`", config=True,)
+
+    allowed_teams = Set(
+        config=True, help="Automatically allow members of selected teams"
     )
 
-    bitbucket_team_whitelist = team_whitelist
+    bitbucket_allowed_teams = allowed_teams
```

Comment:
We don't need to keep the deprecated `bitbucket_` prefix alias for the new trait

```suggestion
```